common/src/main/java/dev/lavalink/youtube/cipher/RemoteCipherManager.java

```diff
@@ -251,7 +264,7 @@ private String getTimestampFromScript(HttpInterface httpInterface, String player
             .end()
             .done();
         request.setEntity(new StringEntity(requestBody, ContentType.APPLICATION_JSON));
-        applyPass(request);
```

Comment:
As an aside and not strictly something to change now/in this PR, I wonder if it would be worth looking into moving all this to the context filter class.

We would still need to set an attribute on the request context so that the filter knows to apply the relevant headers, so it wouldn't necessarily save on any lines unless we can apply that context to the interface, and just have a generic method for returning a cipher server-specific interface for requests, but it seems like the more appropriate place for this.

Food for thought. What do you reckon?

Reply:
We could definitely move it to the request context class. I have seen that more as for more dynamic context stuff in my limited past experience, like the oauth stuff, so it didn't cross my mind at all.

Reply:
Made a context filter just for the remotecipher service and put that on here